### PR TITLE
Dynamically adding rise-storage based on Rise Cache version [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- `main.js` checks RC version and dynamically adds all `rise-storage` dependencies and configures which version of `rise-storage` is used

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: src/widget.html [hunk 1]
```diff
@@ -5,9 +5,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Video Widget</title>
 
-  <script src="components/webcomponentsjs/webcomponents-lite.min.js"></script>
```
Comment (donnapep) [inline src/widget.html]: @stulees Just curious about what problems you experienced with having this script inline?

**Comment by donnapep** [comment]
@stulees Should there be test coverage for this?

**Comment by stulees** [comment]
@donnapep There is test coverage for the other pieces, widget-common RC, rise-storage v2, in their respective repos. I couldn't see a way to test the dynamic addition of specific `rise-storage` version executed by `main.js` prior to anything else executing.

File: src/widget.html (hunk 1 above)
Comment (stulees) [inline src/widget.html]: @donnapep "WebComponentsReady" fires which kicks off our widget in `main.js`. When I left it inline and dynamically added just the components source and element, I was experiencing the RC ping from `rise-storage` failing the try - catch and it halted any further execution of code. By dynamically adding the polyfill right after the component source and element, it seemed to fix it.

**Comment by donnapep** [comment]
@rodrigopavezi What do you think? Is this change important enough to spend some more time trying to figure out test coverage?

**Comment by rodrigopavezi** [comment]
@donnapep @stulees I think it is fine, I don't see also how we could test it. cheers

**Comment by rodrigopavezi** [comment]
@stulees 👍 looks good cheers

**Comment by stulees** [comment]
@rodrigopavezi Please review last commit, same integration tests as Image. Thanks

**Comment by rodrigopavezi** [comment]
@stulees integration tests looks good 👍 cheers

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review last commit, thanks

**Comment by donnapep** [comment]
👍

**Comment by stulees** [comment]
@donnapep Please review last commit, thanks

**Comment by donnapep** [comment]
👍

**Comment by stulees** [comment]
@donnapep Please review last commit, thanks

**Comment by donnapep** [comment]
👍

**Comment by stulees** [comment]
@rodrigopavezi Please review last commit, thanks

**Comment by rodrigopavezi** [comment]
@stulees looks good 👍  cheers

**Pull request merged**